<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRegisterDialog</class>
 <widget class="QDialog" name="QRegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>280</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtParkingPOS - Registro de Usuario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Usuario:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Contraseña:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Confirmar Contraseña:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="userLEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="passLEdit">
         <property name="maxLength">
          <number>8</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passConLEdit">
         <property name="maxLength">
          <number>8</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarPButton">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarPButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>userLEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>aceptarPButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passLEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>aceptarPButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passConLEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>aceptarPButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelarPButton</sender>
   <signal>clicked()</signal>
   <receiver>QRegisterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
